classes/class-admin.php

```diff
@@ -673,6 +673,9 @@ public function wp_ajax_reset() {
 				esc_html__( "You don't have sufficient privileges to do this action.", 'stream' )
 			);
 		}
+		
+		// Create and update the database tables if they do not exist.
+		$this->plugin->install->check();
```

Comment:
Is this check here `defined( 'DOING_AJAX' ) && DOING_AJAX` going to stop this particular call from doing anything?

https://github.com/tomjn/stream/blob/02baea3f526407a0883bee1ccf769562553d012e/classes/class-install.php#L91-L93

Reply:
🤦🏻 I wouldn't have noticed without your help :)

Reply:
Do you think `$this->plugin->install->install( $this->plugin->get_version() );` might make more sense?